schema/assets/image.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Image",
+    "description": "Asset containing an image that can be referenced by layers.",
+    "allOf": [
+        {
+            "$ref": "#/$defs/assets/asset"
+        },
+        {
+            "type": "object",
+            "properties": {
+                "w": {
+                    "title": "Width",
+                    "description": "Width of the image",
+                    "type": "number"
+                },
+                "h": {
+                    "title": "Height",
+                    "description": "Height of the image",
+                    "type": "number"
+                },
+                "p": {
+                    "title": "File Path",
+                    "description": "Path to the image file",
+                    "type": "number"
+                }
```

Comment:
Image assets (and also other asset types that refer to files like audio and data assets) split the path into `p` and `u`. If they refer to a file, `u` is the directory path, `p` the filename.

Reply:
I was curious about that, there may be some inconsistencies around how this is currently used. I'm not sure if Skottie uses the directory path. I also recall some examples where `p` contained both the directory path and filename.

Still good to document and we can work through the inconsistencies later.

Reply:
To be honest I wish it only had one property, but that's how bodymovin exports it